plugins/include/sqlx.inc

```diff
@@ -278,18 +379,24 @@ native SQL_GetQueryString(Handle:query, buffer[], maxlength);
  * result set if one is available.  Otherwise, the current result set is 
  * destroyed and will no longer be accessible.
  *
- * This function will always return false on SQLite, and when using threaded 
- * queries in MySQL.  Nonetheless, it has the same effect of removing the last 
- * result set.
+ * @note This function will always return false on SQLite, and when using threaded
+ *       queries in MySQL.  Nonetheless, it has the same effect of removing the last
+ *       result set.
  *
- * @param query		Query Handle.
- * @return			True on success, false on failure.
+ * @param query         Query Handle.
+ * 
+ * @return              True on success, false on failure.
```

Comment:
Misses `@error` with those possibles errors `Invalid query handle`, `No result set in this query`